packages/back-end/src/services/organizations.ts

```diff
@@ -273,6 +275,59 @@ export async function addMemberToOrg({
   });
 }
 
+export async function addMemberToTeam({
+  organization,
+  userId,
+  teamId,
+}: {
+  organization: OrganizationInterface;
+  userId: string;
+  teamId: string;
+}): Promise<void> {
+  // If member is a pending member, skip
+  if (organization?.pendingMembers?.find((m) => m.id === userId)) {
+    return;
+  }
+
+  const member = organization.members.find((m) => m.id === userId);
+
+  // If member doesn't exist in the org or is already in the team, skip
+  if (!member || member.teams?.includes(teamId)) {
+    return;
+  }
+
+  // Create teams array for member if it does not exist
+  if (!member.teams) {
+    member.teams = [];
+  }
+  member.teams.push(teamId);
+
+  await updateOrganization(organization.id, { members: organization.members });
```

Comment:
It doesn't seem like we're actually updating the organization.members array before we call updateOrganization. We're building the `member.teams` array, but it's disconnected from the `organization.members`. Unless I'm reading it incorrectly.

It looks like this method isn't wired up anywhere, so disregard if this is still a WIP.

Reply:
Since we're working with a reference to the member object when we find it in the org's members array this means that we'll effectively be mutating the member object within the members array. Did a sanity check just to make sure:
![Screenshot 2023-08-23 at 3 58 51 PM](https://github.com/growthbook/growthbook/assets/14947905/00dc6774-79a4-4db5-9cf1-dc7736373fa3)